src/commands/status/docker.js

```diff
@@ -0,0 +1,99 @@
+const { table } = require('table');
+const chalk = require('chalk');
+
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+
+const PRESETS = require('../../presets');
+
+class DockerStatusCommand extends BaseCommand {
```

Comment:
Let’s rename it ”services”

Reply:
Done, command is now `mn status:services`